proposals/forms.py

```diff
@@ -880,7 +880,14 @@ def clean(self):
         from proposals.utils.stepper import Stepper
 
         validator = Stepper(self.instance, request=self.request)
-        if validator.get_form_errors():
+
+        form_errors = validator.get_form_errors()
+
+        if form_errors:
+            # If there are only errors on the submit form, we override this
+            # validation. Otherwise saving the updated data becomes impossible
+            if len(form_errors) == 1 and form_errors[0]["page_name"] == _("Indienen"):
```

Comment:
Works surprisingly well, but I would much rather have a test like

```python3
"proposals/submit" in error["url"].lower()"
```

The URL just feels a lot more dependable.